storage/storageCacherAdapter/storageCacherAdapter.go

```diff
@@ -0,0 +1,243 @@
+package storageCacherAdapter
```

Comment:
can you rename the package to `storagecacheradapter`?  all the packages in the storage package follow this pattern

Reply:
Done.